inc/admin/admin.php

```diff
@@ -131,6 +131,14 @@ function alcatraz_page_options_metabox() {
 		),
 	) );
 
+	// Hide Title.
+	$page_options->add_field( array(
+		'name' => __( 'Hide Page Title?', 'alcatraz' ),
+		'desc' => __( '', 'alcatraz' ),
```

Comment:
There is no need to make an empty string translatable here. This can just be `''`.